extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/security/MtlsAuthenticationMechanism.java

```diff
@@ -76,4 +82,8 @@ public Set<Class<? extends AuthenticationRequest>> getCredentialTypes() {
     public Uni<HttpCredentialTransport> getCredentialTransport(RoutingContext context) {
         return Uni.createFrom().item(new HttpCredentialTransport(HttpCredentialTransport.Type.X509, "X509"));
     }
+
+    public void setRoleMappings(Map<String, Set<String>> roles) {
```

Comment:
Please make this package friendly (it's set in same package) because it makes me really nervous anyone can set roles. Also if it were really public, it would have to consider concurrency. Thanks

Reply:
Sure, I've just removed `public`...

>it makes me really nervous anyone can set roles

Anyone can set them on the identity in any case with augmentors :-) but sure, I agree it is better avoid `public`

Reply:
Yeah, I don't have clear concept how access control should work, but I don't like to give away anything (here `public`) unless I have a good reason. Thanks for change.